app/database/operator/server_data_operator/handlers/channel.ts

```diff
@@ -203,8 +203,8 @@ const ChannelHandler = (superclass: any) => class extends superclass {
         }
 
         const memberships: ChannelMember[] = channelMemberships.map((m) => ({
-            id: `${m.channel_id}-${m.user_id}`,
             ...m,
+            id: `${m.channel_id}-${m.user_id}`,
```

Comment:
We are assigning an id to the membership before here, which was being overwritten due to the order.

We might want to remove the id we are assigning in the previous step, unless it is used for something different (like the myMembership table).

Reply:
can you point me where is that happening ?

Reply:
is it here ? 

`prepareMyChannelsForTeam` ? I guess we can just pass `channelMembers` argument to `handleChannelMembership` instead of `memberships`

Reply:
I guess if we do ☝️ that should fix it entirely. let's make that change in this PR and we should be good to go

Reply:
Fixed in prepareMyChannelsForTeam, but left it here also, since I think it makes sense.